quickwit/quickwit-ingest/src/ingest_v2/ingester.rs

```diff
@@ -1181,6 +1226,16 @@ pub async fn wait_for_ingester_decommission(ingester_opt: Option<IngesterService
     }
 }
 
+struct LocalPersistSubrequest {
+    queue_id: String,
```

Comment:
We have a few types aliais in `quickwit_proto::types:`
```suggestion
    queue_id: QueueId,
```